.gitignore

```diff
@@ -53,4 +53,11 @@ compile_commands.json
 
 # Build Directory 
 Build/
+
+#Shaders other than the test ones
+*.frag
+!fs_test.frag
+*.vert
+!vs_test.vert
+
 .vscode/
```

Comment:
Could you add a section for editor specific files, this should ahve been added in a previous merge but somebody forgot (yes it was me)  😅

Reply:
> 
> 
> This merge is ✨! I've left a comment that should be reviewed before merging!

Hmm can I just make that change, push to gui, and have this pull request update automatically like it did when master changed?

Reply:
Done!

Reply:
Yes, perfect.